docs/management_commands.rst

```diff
@@ -0,0 +1,74 @@
+Django Management Commands
+==========================
+
+Export Command
+--------------
+
+The ``export`` command allows you to export data from a specified Django model
+or a resource class. The exported data can be saved in different formats, such
+as CSV or XLSX.
+
+Usage
+-----
+
+.. code-block:: bash
+
+    python manage.py export <format> <resource> [--encoding ENCODING]
+
+- **format**: Specify the format in which the data should be exported. -
+- **resource**: Specify the resource or model to export. Accepts a resource class or a model class in dotted path format. - **--encoding** (optional): Specify the encoding (e.g., 'utf-8') to be used for the exported data.
+
+Example
+-------
+
+.. code-block:: bash
+
+    python manage.py export CSV auth.User
+
+This command will export the User model data in CSV format using utf-8
+encoding.
+
+Another example:
+
+.. code-block:: bash
+
+    python manage.py export XLSX mymodule.resources.MyResource
+
+This command will export the data from ``MyResource`` resource in XLSX format
+using utf-8 encoding.
+
+Import Command
+--------------
+
+The ``import`` command allows you to import data from a file into a specified
+Django model or a custom resource class.
+
+Usage
+-----
+
+.. code-block:: bash
+
+    python manage.py import <resource> <import_file_name> [--format FORMAT] [--encoding ENCODING] [--dry-run] [--raise-errors]
+
+- **resource**: The resource class or model class in dotted path format.
+- **import_file_name**: The file from which data is imported (``-`` can be used to indicate stdin).
```

Comment:
When I tested this:

```bash
infile=/Users/matthew/Projects/django-import-export/tests/core/exports/200-books.xlsx
./manage.py import core.tests.resources.BookResource --format XLSX - < $infile
```

It crashed with:

```
  File "/Users/matthew/Projects/django-import-export/tests/./../import_export/management/commands/import.py", line 59, in handle
    data = sys.stdin.read()
           ^^^^^^^^^^^^^^^^
  File "<frozen codecs>", line 322, in decode
UnicodeDecodeError: 'utf-8' codec can't decode bytes in position 15-16: invalid continuation byte
```

I also noticed that you have to specify `--format` if importing from stdin.